install/kubernetes/helm/istio/charts/pilot/templates/deployment.yaml

```diff
@@ -52,6 +52,7 @@ spec:
           imagePullPolicy: {{ .Values.global.imagePullPolicy }}
           args:
           - "discovery"
+          - --registries={{ .Values.registries | join "," }}
```

Comment:
Would it be possible to not add this - instead use the mesh config list of MCP servers, or add an option there ?
The current plan is to have mesh configuration defined in the API - and we already have the list of config sources there,

Reply:
Today in pilot list of MCP servers is retrieved via `meshconfig.Configsources`, this PR also enables retrieval of SSE MCP servers via an environment variable. We also allow users to configure registries via pilot's command line argument. I can consolidate these configs as MCP config in mesh config, however would it be ok if I created an issue and did it in the follow up PR?